backend/src/prisma/recipes.js

```diff
@@ -22,8 +22,21 @@ async function updateRecipe() {
 }
 
 
-async function deleteRecipe() {
-    return null
+async function deleteRecipe(id) {
+    try {
+        await prisma.recipe.delete({
+            where: {
+                id: id,
+            },
+        })
+        await prisma.recipeIngredient.updateMany({
+            where: {
+                recipeId: id,
+            },
+        })
+    } catch (e) {
+        throw e
```

Comment:
We need to catch specific DB error here. Something similar to https://github.com/benkeanna/MixBook/blob/main/backend/src/prisma/ingredients.js#L46.

Reply:
Still needed